sklearn/tree/tests/test_tree.py

```diff
@@ -503,6 +504,13 @@ def test_error():
 
     for name, TreeEstimator in ALL_TREES.items():
         # Invalid values for parameters
+        if name == "DecisionTreeRegressor" or name == "DecisionTreeClassifier":
```

Comment:
I think it will also work for ExtraTreeClassifier/ExtraTreeRegressor, so something like
`if name not in ["Presort-DecisionTreeClassifier", "Presort-DecisionTreeRegressor"]` might be better.
Or maybe we can simply remove this line since we're also doing duplicate check in other situations below.

Reply:
ExtraTreeRegressor and ExtraTreeClassifier doesn't seem have a presort argument. So test will throw an error when TreeEsimator either of the above. Thats the reason I put the line.

Reply:
@Fenil3510 Thanks :) Sorry that I posted the review without confirming carefully.